components/higher-order/with-focus-return/index.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
What about `utilities` instead of higher-order? Alternatively, we can just keep them in the root. cc @aduth @youknowriad